### PR TITLE
Install bitnami-labs/sealed-secrets

### PR DESCRIPTION
**Pull request by teodor-pripoae**

### CODE REVIEW

**Comment by teodor-pripoae** [comment]
I will modify the `SealedSecrets` struct after `https://github.com/flanksource/commons/pull/12/files` is merged.

```go
type SealedSecrets struct {
	Enabled    bool   `yaml:"enabled,omitempty"`
        Certificate certs.Certificate `yaml:"certificate,omitempty"`
}
```

File: test/common.yml [hunk 1]
```diff
@@ -125,3 +125,9 @@ ca:
   cert: .certs/ingress-ca-crt.pem
   privateKey: .certs/ingress-ca-key.pem
   password: foobar
+sealedSecrets:
+  enabled: true
```
Comment (moshloop) [inline test/common.yml]: Switch from enabled ==> disabled and add a version config

File: go.mod [hunk 2]
```diff
@@ -4,10 +4,11 @@ go 1.12
 
 require (
 	github.com/aws/aws-sdk-go v1.29.5
+	github.com/bitnami-labs/sealed-secrets v0.10.0
```
Comment (moshloop) [inline go.mod]: do we need this?

**Comment by moshloop** [comment]
Can we add docs and an e2e test ?

**Comment by teodor-pripoae** [comment]
Done

**Pull request merged**